helfi_features/helfi_events/assets/js/eventList.js

```diff
@@ -34,32 +34,32 @@
         // Base element for event, wihout text elements from api
         const eventElement = $(`
           <div class="event-list__event">
-            <a class="event-list__events-container" href="${drupalSettings.helfi_events.baseUrl}/events/${event.id}" aria-label="(${externalLink})">
-              <div class="event-list__image-container">
-                <div class="event-list__tags event-list__tags--mobile" role="Region" aria-label="${eventKeywords}">
-                </div>
+            <div class="event-list__image-container">
```

Comment:
btw, do we get the photographer data from API? If we do, we could add it to the img-element as `data-photographer="John Smith"`